packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -131,6 +131,18 @@ suite('createComponent', () => {
     assert.equal(el.textContent, 'Component without event map.');
   });
 
+  /*
+    The following test is a type-only test.
+  */
+  test('renders element with expected type', async () => {
+    type TypedComponent = ReactWebComponent<BasicElement>;
+
+    let TypedBasicElement!: TypedComponent;
+
+    // @ts-expect-error
+    <TypedBasicElement bool={"string"}></TypedBasicElement>
```

Comment:
Add a comment explaining why this should be a type error.

Reply:
done!